composer.json

```
@@ -13,7 +13,7 @@
         "php-http/httplug-bundle": "^1.19",
         "sylius/sylius": "^1.10",
         "symfony/property-info": "^4.4|^5.4",
-        "symfony/serializer": "^4.4|^5.4",
+        "symfony/serializer": "^6.0",
         "symfony/lock": "^4.4|^5.4",
         "symfony/messenger": "^4.4|^5.4"
```

Since we are dropping serializer <6 support, we also need to drop other symfony 4 component support as well ser serializer 6+ does not support symfony 4.4 anymore.

```suggestion
        "symfony/lock": "^5.4|^6.0",
        "symfony/messenger": "^5.4|^6.0"
```